docs/automation/web.md

```diff
@@ -310,3 +310,144 @@ component.assertUIObjectNotPresent();   // equals to Assert.assertTrue(!componen
 ```
 >Dynamic elements loading. 
 **waitForJSToLoad()** method was introduced in AbstractPage class. It uses JS under the hood and helps to wait till all dynamic web elements on the page are getting loaded.
```

Comment:
It uses JS under the hood and helps to wait till all **the** dynamic web elements on the page **are loaded.**